store/dataStore.test.ts

```diff
@@ -15,21 +26,68 @@ describe('Data Store', () => {
     const { result } = renderHook(() => useDataStore());
 
     act(() => {
-      result.current.updateUser(userId, userEmail);
+      result.current.updateUser(userData.userId, userData.userEmail);
     });
 
-    expect(result.current.user.id).toBe(userId);
-    expect(result.current.user.email).toBe(userEmail);
+    expect(result.current.user.id).toBe(userData.userId);
+    expect(result.current.user.email).toBe(userData.userEmail);
   });
   it('Checks the reset user state matches default', () => {
     const { result } = renderHook(() => useDataStore());
 
     act(() => {
-      result.current.updateUser(userId, userEmail);
+      result.current.updateUser(userData.userId, userData.userEmail);
       result.current.resetUser();
     });
 
     expect(result.current.user.id).toBe('');
     expect(result.current.user.email).toBe('');
   });
+  it('Check the default NFL Teams state', () => {
+    const { result } = renderHook(() => useDataStore());
+    expect(result.current.NFLTeam.teamName).toBe('');
+    expect(result.current.NFLTeam.teamLogo).toBe('');
+  });
+  it('Check the updated NFL Teams state', () => {
+    const { result } = renderHook(() => useDataStore());
+
+    act(() => {
+      result.current.updateNFLTeam(gameData);
+    });
+
+    expect(result.current.NFLTeam.teamName).toBe(gameData.teamName);
```

Comment:
Q: Why are using the Jest toBe() method here instead of the toEqual() method?

Reply:
That's a great question. 

I used `toBe()` to test for strict equality for actual value to match expected value. 

Since the gameData is an object, you can use `toEqual()`. I decided to opt for `toBe()` to be concise with other tests.